packages/cli/src/resource/commands.json

```diff
@@ -158,7 +158,7 @@
     "success": "Upgrade project successfully."
   },
   "validate": {
-    "description": "Validate the Microsoft Teams app using manifest schema or validation rules."
+    "description": "Validate the Microsoft Teams app using manifest schema, validation rules or test cases."
```

Comment:
Add comma before 'or' as we follow American English, i.e. ....validation rules, or test cases.

Reply:
Thanks @supkasar , comment resolved.